src/content/guides/code-splitting.md

```diff
@@ -48,7 +49,7 @@ There are three general approaches to code splitting available:
 
 ## Entry Points
 
-This is by far the easiest, and most intuitive, way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
+This is by far the easiest and most intuitive way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
```

Comment:
I think there has to be a comma, but after and before `by far`. Any grammar experts around?

Reply:
@EugeneHlushko yes, I also think after `by far` it makes sense, but then we must have it after `intuitive` as well.

Reply:
```suggestion
This is by far, the easiest and the most intuitive way to split code. However, it is more manual and has some pitfalls we will go over. Let's take a look at how we might split another module from the main bundle:
```